pkg/services/grn/doc.go

```diff
@@ -0,0 +1,19 @@
+// package GRN provides utilities for working with Grafana Resource Names
+// (GRNs).
+
+// A GRN is an identifier which encodes all data necessary to retrieve a given
+// resource from its respective service.
+
+// A GRN string is expressed in the format:
+//   grn:${service}:${tenant_id}:${org_id}:${identifier}
+//
+// The format of the final identifier is defined by the owning service and not
+// validated by the GRN parser. We recommend the format ${kind}/${uid}.
+
+// Individual segments may be omitted from a GRN if the information is not
+// relevant for the given resource. A segment is omitted by leaving the value
+// blank. For example, resource which aren’t associated with a tenant id would
+// omit that segment:
+//   grn:${service}::${org_id}:${identifier}
+
+package grn
```

Comment:
is this service?  or models/util?


long term, I *think* this living in the plugins SDK makes the most sense, but short term anywhere in grafana core is good

Reply:
I do agree with you on where it should live; this was me being lazy about getting it somewhere fast so we could have this discussion :) 

If you're asking about where I put this package inside grafana, it was also a lazy choice - this should be a library in the SDK.